app/models/user.rb

```diff
@@ -209,6 +216,9 @@ def validate_pin_pattern
     if pin && pin&.scan(/(.)\1{2,}/)&.empty? && pin.scan(/(..)\1{1,}/)&.empty? == true
       true
     else
+      msg = 'PIN does not meet our requirements. Please try again.'
+      return errors.add(:pin, msg) unless MlnConfigurationController.new.feature_flag_config('signup.pin_password.enabled') 
```

Comment:
I written condition block, Rubocop complained about it. After that i written like this